src/wit_world/__init__.py

```diff
@@ -0,0 +1,13 @@
+from typing import TypeVar, Generic, Union, Optional, Protocol, Tuple, List, Any, Self
```

Comment:
This is generated by the wasi-tools/regenerate_bindings.sh script and excluded from packaging. At runtime this will be provided by componentize-py on wasi